Example/Tests/XExtensionItemParametersTests.m

```diff
@@ -4,6 +4,14 @@
 #import "CustomParameters.h"
 #import "XExtensionItem.h"
 
+#define XCTAssertEqualExtensionItems(item1, item2) \
```

Comment:
Might be odd to use a prefix that’s not ours because theoretically Apple could add this same function.